fastlane/Fastfile

```diff
@@ -376,45 +377,46 @@ end
       }
     )
 
-    sh("mv ../build/Simplenote.ipa \"../build/Simplenote Internal.ipa\"")
+    File.rename(File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote.ipa'), File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote Internal.ipa'))
 
     appcenter_upload(
       api_token: ENV["APPCENTER_API_TOKEN"],
       owner_name: "automattic",
       owner_type: "organization",
       app_name: "Simplenote",
-      file: "./build/Simplenote Internal.ipa",
+      file: File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote Internal.ipa'),
       notify_testers: false
     )
 
     sentry_upload_dsym(
       auth_token: ENV["SENTRY_AUTH_TOKEN"],
       org_slug: 'a8c',
       project_slug: 'simplenote-ios',
-      dsym_path: "./build/Simplenote.app.dSYM.zip"
+      dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
   end
 
   #####################################################################################
-  # build_and_upload_itc
+  # build_and_upload_to_app_store_connect
   # -----------------------------------------------------------------------------------
   # This lane builds the app and upload it for external distribution
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane build_and_upload_itc [skip_confirm:<skip confirm>] [create_release:<Create release on GH>] [beta_release:<intermediate beta?>]
+  # bundle exec fastlane build_and_upload_to_app_store_connect [skip_confirm:<skip confirm>] [create_release:<Create release on GH>] [beta_release:<intermediate beta?>]
   #
   # Example:
-  # bundle exec fastlane build_and_upload_itc
-  # bundle exec fastlane build_and_upload_itc skip_confirm:true
+  # bundle exec fastlane build_and_upload_to_app_store_connect
+  # bundle exec fastlane build_and_upload_to_app_store_connect skip_confirm:true
   #####################################################################################
```

Comment:
No need to address this in the context of this PR, but FYI, while working on WordPress iOS, we agreed to stop using this verbose documentation in favor of a more lean approach.

https://github.com/wordpress-mobile/WordPress-iOS/commit/8b47d7ce5c17461ff6dc56aa779f7846e4359d89

Reply:
Good to know! I'll make those changes in a separate PR.

Reply:
Yeah, no rush.